cg/services/deliver_files/delivery_file_fetcher_service/error_handling.py

```diff
@@ -21,3 +24,19 @@ def wrapper(*args, **kwargs):
             return None
 
     return wrapper
+
+
+def handle_validation_errors(func):
+    """
+    Log an error when a validation error occurs.
+
+    """
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except ValidationError as error:
+            raise NoDeliveryFilesError(str(error))
+
+    return wrapper
```

Comment:
Unless all validation errors raised within the deliver file service should result in the NoDeliveryFilesError, we should name this a bit more specific. Now if there is are no case files or no delivery_data when trying to instantiate the DeliveryFiles object, it will still show up as a NoDeliveryFilesError

Reply:
Wait, I am stupid - we are only trying to instantiate the DeliveryFiles BaseModel? If so, please resolve